analysis/01_meetup_api_discovery/new_contributor_onboarding/JulianaLam/MeetUp_Info.py

```diff
@@ -0,0 +1,11 @@
+'''
+Juliana Lam
+Meet-Up API Info
+'''
+
+import meetup.api
+
+url = "https://api.meetup.com/"
+apikey = "5f5c323971225b60745a3411816a70"
```

Comment:
Please remove your API key! You don't want this getting committed to a public place :)